pkgs/development/compilers/flutter/wrapper.nix

```diff
@@ -35,9 +37,25 @@
 let
   # By default, Flutter stores downloaded files (such as the Pub cache) in the SDK directory.
```

Comment:
Should this comment be moved down to line 56?

Reply:
right, done